Appsilon/mbaza
# Jakub refactor styling 237

Closes #237 

### Description
- applied PascalCase everywhere to name components files
- imported scss modules as styles everywhere
- used camelCase when defining scss modules classes
- moved inline styles to scss modules
- used @use to import sass modules instead of @import
- removed app.global.css
- refactored global scss modules

### How to test
- check code structure, quality etc.
- click through the app to detect if there's no missing styles/mismatches etc
